web/src/js/ducks/flows.js

```diff
@@ -144,15 +144,21 @@ export function upload(file) {
     return dispatch => fetchApi('/flows/dump', { method: 'post', body })
 }
 
-
-export function select(id) {
-    return {
-        type: SELECT,
-        flowIds: id ? [id] : []
+/**
+ * @public
+ */
+export function select(id, selectedFlowIds, ctrl) {
```

Comment:
@mhils `selectedFlowIds` and `ctrl` are optional here, you can ignore them if you don't want to select multiple flows ;)